ui/pages/confirmations/hooks/useCurrentSignatureSecurityAlertResponse.test.ts

```diff
@@ -22,25 +22,16 @@ const currentConfirmationMock = {
 };
 
 const getMockCurrentState = () =>
-  getMockConfirmState({
-    metamask: {
-      unapprovedPersonalMsgs: {
-        '1': { ...currentConfirmationMock, msgParams: {} },
-      },
-      pendingApprovals: {
-        '1': {
-          ...currentConfirmationMock,
-          origin: 'origin',
-          requestData: {},
-          requestState: null,
-          expectsResult: false,
+  getMockPersonalSignConfirmStateForRequest(
+    currentConfirmationMock as unknown as SignatureRequestType,
```

Comment:
rather than using `as unkown as SignatureRequestType`, we can use TransactionType instead of SignatureType for the type value. This is the type that SignatureRequestType specifies 

```
import { TransactionType } from '@metamask/transaction-controller';

// ...

const currentConfirmationMock = {
  id: '1',
  status: 'unapproved',
  time: new Date().getTime(),
  type: TransactionType.personalSign,
  securityAlertResponse: mockSecurityAlertResponse,
} as SignatureRequestType;
```

```suggestion
    currentConfirmationMock,
```